city_scrapers/spiders/bisnd_bps.py

```diff
@@ -0,0 +1,158 @@
+from scrapy import Selector
+
+
+class BisndBpsSpider(CityScrapersSpider):
+    name = "bisnd_bps"
+    agency = "Bismarck Public Schools"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.bismarckschools.org/Page/401"]
+
+    """
+    Meetings' time and location is taken from organization's following webpage:
+    https://www.bismarckschools.org/Page/398
+    """
+    meeting_time = time(17, 15)
+    location = {
+        "name": "Tom Baker Meeting Room of the City/County Office Building",
+        "address": "221 N Fifth Street, Bismarck, ND",
+    }
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+        """
+
+        extracted_input = response.css(
+            '.ui-widget-detail input[type="hidden"]::attr(value)'
+        ).extract_first()
+
+        parsable_data = self._parsed_data(extracted_input)
+
+        for item in parsable_data:
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description="",
+                classification=self._parse_classification(item),
+                start=self._parse_start(item),
+                end=None,
+                all_day=False,
+                time_notes=self._parse_time_notes(item),
+                location=self.location,
+                links=self._parse_links(item),
+                source=response.url,
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        selected_title = item[3]
+        html_title_str = lhtml.fromstring(selected_title)
+        title = html_title_str.text_content().strip()
+        return title if title is not None else ""
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        title = "".join(item[3].split(">")[1].split("<")[:-1])
+        return COMMITTEE if "committee" in title.lower() else BOARD
+
+    def _parse_start(self, item):
+        """Parse start datetime as a naive datetime object."""
+        extracted_date = [
+            html.unescape("".join(element.split(">")[1].split("<")[:-1]))
+            for element in item[:3]
+        ]
+        result = " ".join(extracted_date)
+        parsed_date = result if "(" not in result else result.split(" (")[0]
+        date_obj = datetime.strptime(parsed_date, "%Y %B %d")
+
+        return datetime.combine(date_obj, self.meeting_time)
+
+    def _parse_time_notes(self, item):
+        """
+        Meeting detail for non 'Regular' type meetings
+        are accessable from the meeting agenda.
+        The notes are added to further clarify the meeting details.
+        """
+        return "Meetings that are not of type 'Regular' are held at specific locations with specific timing, please refer to the meeting agenda for more details."  # noqa
```

Comment:
**suggestion (blocking):** In this case, I would suggest modifying the behavior of this method so that `if` "regular" is not included in the meeting title, `time_notes` is assigned this note, otherwise the value is an empty string. That would ensure we're only including the note on the meeting items where it's relevant.

Reply:
That's definitely a better approach, will make the change.